src/main/java/network/brightspots/rcv/ClearBallotCvrReader.java

```diff
@@ -7,6 +7,15 @@
  * file, You can obtain one at https://mozilla.org/MPL/2.0/.
  */
 
+/*
+ * Purpose: parses Clear Ballot cvr files into CastVoteRecords
+ * Design: Clear Ballot data is stored in .csv files one row per csv.  This class uses a buffered
```

Comment:
looks like there are 12 instances of a period followed by two spaces, and 10 instances of a period followed by a single space. can we pick one?

Reply:
Please pick one space! 🙏

Reply:
Love the enthusiasm here.  I wish there was more enthusiasm for actually writing comments :)  Anyhow, I went through the relevant style docs and this is not addressed.  So I did some light reading on this highly "divisive" subject.   In source code (which is typically rendered in mono-spaced fonts so that stuff lines up vertically) I found no clear consensus.  (In English text the consensus is very much single space, since modern font renderers are proportional and are designed to look good with single space.)  So I made the module header comments consistent with double spaces after sentences.  That's just how I've always done it and it looks more readable to me. 

That said, I am totally fine if someone else wants to pick a standard, tell people about it, make the code base consistent, etc.. but that is outside the scope of this PR.

Reply:
Ha, yeah, I definitely prefer a single space, but we can certainly leave that for another PR.